### PR TITLE
docs(expect): minor doc tweaks

### PR DESCRIPTION
**Pull request by iuioiua**

### CODE REVIEW

File: expect/fn.ts [hunk 1]
```diff
@@ -19,14 +17,16 @@
  *   expect(mockFn).toHaveBeenCalledTimes(1);
  * });
  * ```
+ *
+ * @module
```
Comment (iuioiua) [inline expect/fn.ts:21]: Fixes module docs not being displayed.

File: expect/mod.ts [hunk 2]
```diff
@@ -5,6 +5,16 @@
 /**
  * This module provides Jest compatible expect assertion functionality.
  *
+ * ```ts no-assert
+ * import { expect } from "@std/expect";
+ *
+ * const x = 6 * 7;
+ * expect(x).toEqual(42);
+ * expect(x).not.toEqual(0);
+ *
```
Comment (iuioiua) [inline expect/mod.ts:14]: Aligns with other packages which usually have an example first.

File: expect/fn.ts (hunk 1 above)
Comment (kt3k) [reply expect/fn.ts:21]: Good catch

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**